src/inmanta/execute/scheduler.py

```diff
@@ -55,7 +55,7 @@
 LOGGER = logging.getLogger(__name__)
 
 
-MAX_ITERATIONS = 10000
+MAX_ITERATIONS = 100000
```

Comment:
```suggestion
MAX_ITERATIONS = 100_000
```
Slightly easier to count.